### PR TITLE
Added vue linting

### PR DESCRIPTION
**Pull request by jconway325**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,7 +67,8 @@
     "eslint-plugin-react-hooks": "4.2.0",
     "object.assign": "^4.1.2",
     "object.entries": "^1.1.2",
-    "prettier": ">=2.0.0"
+    "prettier": ">=2.0.0",
```
Comment (BenBrewerBowman) [inline package.json]: This is my fault, but can we remove this?

File: vue.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    ecmaFeatures: {
+      globalReturn: false,
+      impliedStrict: false,
+      jsx: false
+    },
+    parser: "@typescript-eslint/parser"
+  },
+  rules: {
+    "vue/html-closing-bracket-newline": "off",
+    "vue/html-closing-bracket-spacing": "off",
```
Comment (BenBrewerBowman) [inline vue.js]: Why are these rules being overridden?

**Comment by BenBrewerBowman** [comment]
While you're in here, can you add this to the base config?
`"no-debugger": "warn",`

File: vue.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    browser: true,
+  },
+  parser: "vue-eslint-parser",
+  parserOptions: {
+    ecmaFeatures: {
+      globalReturn: false,
+      impliedStrict: false,
+      jsx: false
+    },
+    parser: "@typescript-eslint/parser"
```
Comment (BenBrewerBowman) [inline vue.js]: I think you are declaring 2 diff parsers here, no? 
`parser: "vue-eslint-parser"`
`parser: "@typescript-eslint/parser"`

File: vue.js (hunk 2 above)
Comment (BenBrewerBowman) [reply vue.js]: Prettier should take care of these, no?

**Comment by BenBrewerBowman** [comment]
Also can you update the readme w/ installation instructions for a vue project?

File: vue.js (hunk 2 above)
Comment (jconway325) [reply vue.js]: OK, I think we can remove them. Thanks for pointing out that

File: vue.js (hunk 3 above)
Comment (jconway325) [reply vue.js]: I think we can remove the `@typescript-eslint/parser`

**Review by BenBrewerBowman — APPROVED** [review]
Looks great 😻

**Pull request merged**